bebop2/examples/take_off.rs

```diff
@@ -0,0 +1,16 @@
+use std::error::Error;
+use std::net::IpAddr;
+
+use bebop2::Bebop2;
+
+fn main() -> Result<(), Box<dyn Error>> {
+    // let drone_address = "10.202.0.254".parse::<IpAddr>()?;
+    let drone_address = "10.202.0.1".parse::<IpAddr>()?;
```

Comment:
seems to be the correct IP when using sphinx, but the communication is not ok :D